daemon/cmd/daemon.go

```diff
@@ -425,6 +428,17 @@ func NewDaemon(ctx context.Context, epMgr *endpointmanager.EndpointManager, dp d
 	}
 	d.policyTrigger = t
 
+	t, err = trigger.NewTrigger(trigger.Parameters{
+		Name:            "datapath_regeneration",
+		MetricsObserver: &policyTriggerMetrics{},
+		MinInterval:     option.Config.PolicyTriggerInterval,
+		TriggerFunc:     d.datapathRegen,
+	})
+	if err != nil {
+		return nil, nil, err
+	}
+	d.datapathRegenTrigger = t
```

Comment:
It seems that we are reusing the same `t` variable. Wouldn't that mess with the variable that's already set to the previous `t`?

Reply:
At this point, `t` should already be assigned to `d.policyTrigger` so I think it would be safe to change `t`. I'll use a new name to make it clearer.